python/lsst/ip/isr/measureCrosstalk.py

```diff
@@ -316,17 +358,193 @@ def reduce(self, ratioList):
                 else:
                     result = np.concatenate([vv for vv in values if len(vv) > 0])
             ratios[ii][jj] = result
-        coeff, coeffErr, coeffNum = measureCrosstalkCoefficients(ratios, self.config.rejIter,
-                                                                 self.config.rejSigma)
+            self.debugRatios('reduce', ratios, ii, jj)
```

Comment:
And again. I'm sure I must be missing what makes this work though, so sorry about all these.